src/main/index.js

```diff
@@ -122,17 +122,22 @@ function init () {
 }
 
 function delayedInit () {
+  if (app.isQuitting) return
+
   const announcement = require('./announcement')
   const dock = require('./dock')
-  const tray = require('./tray')
   const updater = require('./updater')
   const userTasks = require('./user-tasks')
 
   announcement.init()
   dock.init()
-  tray.init()
   updater.init()
   userTasks.init()
+
+  if (process.platform !== 'darwin') {
+    const tray = require('./tray')
```

Comment:
If you’re doing that, maybe you might want to only load `./user-tasks.js` on Windows only?